INN/umbrella-caribbean
# Homepage right social section

## Changes

This pull request makes the following changes:

- Makes sure the social section on the homepage right section looks ok

![Screen Shot 2019-12-04 at 1 16 02 PM](https://user-images.githubusercontent.com/18353636/70169555-123cc000-1699-11ea-967c-72de6ceccc79.png)

## Why



For #8

## Testing/Questions

Features that this PR affects:

- Homepage right section follow icons


Questions that need to be answered before merging:

- [x] Would it be beneficial to add an arbitrary text field to the Largo Follow widget so we don't have to create custom HTML widgets for areas where they want text above the icons?

Steps to test this PR:

1. Add a custom HTML widget to the homepage right widget area and paste this in:
```
<p>Start a conversation or follow us on social media</p>

<ul class="social-icons circled-icons icons-center">
 	<li><a href="#"><span class="dashicons dashicons-facebook-alt"></span></a></li>
 	<li><a href="#"><span class="dashicons dashicons-twitter"></span></a></li>
 	<li><a href="#"><span class="dashicons dashicons-instagram"></span></a></li>
</ul>
<br/>
```
2. Verify it looks ok
